### ISSUE TITLE
[BUG] Downloaded Games do not launch

### ISSUE THREAD

**Issue by BruninDoMine** [hydralauncher/hydra#1368]
### Describe the bug

Since update 3.1.4, everytime I click on the "play" button, the game does not start.

### Steps to Reproduce

1. Update Hydra to version 3.1.4
2. Install any game
3. Run any game

### Expected behavior

In any previous version, pressing the button would launch the game, but unfortunately, since the last update, it has not happened anymore.

### Additional information and data

[error.txt](https://github.com/user-attachments/files/18282975/error.txt)

### Operating System

Windows 10

### Hydra Version

3.1.4

### Before opening this Issue

- [X] I have searched the issues of this repository and believe that this is not a duplicate.
- [X] I am aware that Hydra team does not offer any support or help regarding the downloaded games.
- [x] I have read the [Frequently Asked Questions (FAQ)](https://github.com/hydralauncher/hydra/wiki/FAQ).

**Comment by zamitto** [comment]
I assume it is happening for any game you try.
Did the same games with the same exact executables worked before 3.1.4?
The logs says Hydra could not run the executable because of permission denied (EACCES)
Any chance you did anything different with Hydra since 3.1.4 update?
Could you try running Hydra with admin mode and try launching a game?

**Comment by BruninDoMine** [comment]
It indeed happens with any game, and I've been using the same executables for a long time, but they only stopped working since the update. No configs were changed since the update, and I have already tried running it as admin. Thanks for the attention. I also found that downgrading Hydra doesn't fix the issue, so something happened when I updated, that botched some kind of authorization the application had.
